src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx

```diff
@@ -119,6 +119,5 @@ or
 
 Tips:
 
-* The folder/s you specify must be at the root level of the docs directory. This doesn't include any subdirectories. For example, `browser` is a valid folder, but `browser-monitoring` is not because it's a subdirectory of `browser`.
-
+* The folder(s) you specify must be at the root level of the `/src/content/docs/` directory. Tis doesn't include any subdirectories. For example, `browser` is a valid folder, but `browser-monitoring` is not because it's a subdirectory of `browser`.
```

Comment:
```suggestion
* The folder(s) you specify must be at the root level of the `/src/content/docs/` directory. This doesn't include any subdirectories. For example, `browser` is a valid folder, but `browser-monitoring` is not because it's a subdirectory of `browser`.
```